src/main/java/picard/illumina/parser/TileMetricsUtil.java

```diff
@@ -163,7 +152,7 @@ public static File renderTileMetricsFileFromBasecallingDirectory(final File illu
                                                     final ValidationStringency validationStringency)
             throws FileNotFoundException {
```

Comment:
Not yours, but I don't think this method throws FileNotFoundException, nor does it use validationStringency.